packages/flutter/test/ui/rrect_test.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/painting.dart';
+
+import 'package:test/test.dart';
+
+void main() {
+  test("RRect.contains()", () {
+    RRect rrect = new RRect.fromRectCustom(
```

Comment:
yeah, looking at this I really think we need a better name than fromRectCustom.

Maybe fromRectAndCorners and then use named arguments for the corners?